addons/web/static/src/legacy/scss/form_view.scss

```diff
@@ -1092,3 +1092,8 @@ $o-form-label-margin-right: 0px;
         }
     }
 }
+
+// No events
+.o_form_view .oe_button_box > .btn.oe_stat_button.no_events{
+    pointer-events: none;
+}
```

Comment:
missing line return

Reply:
odoo classnames must begin with o_ or o-. But do we really need the classname here? couldn't you apply the rule on disabled stat buttons?

Reply:
@rdeodoo I have resolved all the comments that you have mentioned.